go.mod

```diff
@@ -20,7 +20,6 @@ require (
 	k8s.io/apimachinery v0.18.4
 	k8s.io/client-go v12.0.0+incompatible
 	sigs.k8s.io/controller-runtime v0.6.1
-	sigs.k8s.io/controller-tools v0.3.0 // indirect
```

Comment:
This is an indirect dependency required for codegen, could you add this back. Sadly this breaks with `go mod tidy`. Need a better way to handle this. Can you make sure to run `make generate-crds` before committing?

Reply:
Done!